### DISCUSSION TITLE
Version 3.0.2 is broken here

### DISCUSSION THREAD

**Discussion by pedrozimmermannsoares** [noah-nuebling/mac-mouse-fix#870] [Other]
I had to install 3.0.1 here, because this new version didn't open on my mac. A banner "The application "Mac Mouse Fix"
can't be opened." appear and functions doesn't work

Specs:
MacBook Pro
14-inch, 2021

Chip Apple M1 Pro
Memory 16 GB
macOS Sonoma 14.4

![CleanShot 2024-03-05 at 16 28 43](https://github.com/noah-nuebling/mac-mouse-fix/assets/40131294/0696b56a-bdfd-4e43-b3f1-06ee4184d1ef)

**Comment by noah-nuebling** [comment]
Hi there @rotivordep, thanks for the feedback!

Does it work if you try to download it again now? I re-exported the app from Xcode in a different way. 2 people in https://github.com/noah-nuebling/mac-mouse-fix/issues/871 had a similar issue and the re-export seems to have resolved it for them. 

I'll close this now, assuming that the issue has been resolved, please let me know if you still experience any problems.

**Reply by pedrozimmermannsoares** [reply]
It worked now! Thank you! This app is amazing!
